network/src/peers/peers.rs

```diff
@@ -142,42 +143,33 @@ impl<S: Storage + Send + Sync + 'static> Node<S> {
     }
 
     ///
-    /// Broadcasts a connection request to all default bootnodes of the network.
+    /// Broadcasts a connection request to all the supplied peers.
     ///
-    /// This function attempts to reconnect this node server with any bootnode peer
-    /// that this node may have failed to connect to.
-    ///
-    /// This function filters out any bootnode peers the node server is
+    /// This function filters out any peers the node server is
     /// either connnecting to or already connected to.
     ///
-    async fn connect_to_bootnodes(&self) {
+    async fn connect_to_peers(&self, peers: Arc<Vec<SocketAddr>>) {
```

Comment:
```suggestion
    async fn connect_to_addresses(&self, addrs: &[SocketAddr]) {
```
nit: we already have a `Peers` object, so it's better not to mix them up; also, it would be preferable to pass a slice as an argument to this function if it's possible (I see `copied` being used, so it should work)